backend/controllers/users.js

```diff
@@ -9,6 +9,63 @@ const {
   Word,
 } = require("../models");
 
+const _getActivities = async (idList) => {
```

Comment:
- please remove underscore. make it simple camelcase (apply to all related) (ATR)